michelson.md

```diff
@@ -1711,10 +1713,17 @@ value is invalid.
 
   syntax FailedStack ::= #FailureFromMutezValue(Mutez, Int, Int) [function]
   // ----------------------------------------------------------------------
-  rule #FailureFromMutezValue(#Mutez(I), I1, I2)
-    => ( MutezOverflow I1 I2 ) requires I >=Int #MutezOverflowLimit
-  rule #FailureFromMutezValue(#Mutez(I), I1, I2)
-    => ( MutezUnderflow I1 I2 ) requires I <Int 0
+  rule #FailureFromMutezValue(#Mutez(I), I1, I2) => ( MutezOverflow I1 I2 )  requires I >=Int #MutezOverflowLimit
+  rule #FailureFromMutezValue(#Mutez(I), I1, I2) => ( MutezUnderflow I1 I2 ) requires I  <Int 0
+```
+
+```symbolic
+  rule #FailureFromMutezValue(#Mutez(I), I1, I2) => ( MutezOverflow I1 I2 )  requires I >=Int #MutezOverflowLimit [simplification]
+  rule #FailureFromMutezValue(#Mutez(I), I1, I2) => ( MutezUnderflow I1 I2 ) requires I  <Int 0                   [simplification]
```

Comment:
```suggestion
  rule #FailureFromMutezValue(#Mutez(I), I1, I2) => ( MutezOverflow I1 I2  ) requires I >=Int #MutezOverflowLimit [simplification]
  rule #FailureFromMutezValue(#Mutez(I), I1, I2) => ( MutezUnderflow I1 I2 ) requires I  <Int 0                   [simplification]
```

Reply:
@ehildenb Do you know if it is necessary to add these `simplification` rules in a ` ```symbolic ` block? Could we just add it as an annotation to the regular rule above?

Reply:
For the first two simplification rules, it could be placed in either ````symbolic` or ````k`. The rules would still need to be duplicated. The third rule uses `#Bottom` which is not supported by LLVM, so it needs to go into `symbolic`.
In general, I think it makes more sense to keep the rules in `symbolic` because they aren't used by the concrete backend.